docs/developer-guide/en/building.md

```diff
@@ -77,8 +77,8 @@ Kapua is running CI builds against two public environments:
 
 Please be sure that both environments are "green" (i.e. all tests pass) after you commit any changes into `develop` branch.
 
-We also use CI server sponsored by [Red Hat](https://www.redhat.com/en) to automatically push latest Docker images to 
-[Kapua DockerHub account](https://hub.docker.com/r/kapua/). Red Hat CI server checks for code changes every 15 minutes and pushes updated version
+We also use CI server sponsored by [Red Hat](https://www.redhat.com/en) to automatically push the latest Docker images to 
+[Kapua DockerHub account](https://hub.docker.com/r/kapua/). Red Hat CI server checks for code changes every 15 minutes and pushes updated versions
```

Comment:
Hey @geetha-rangaswamaiah,
these two words you have changed, "the" and "versions", are not ok or is rather gramatically wrong. 

Also there are many more spelling errors/gramatic errors in the text, please fix them also. 

Another thing is to sign the Eclipse[ ECA in order to be able to contribute](https://accounts.eclipse.org/user/login?destination=user/eca).

Reply:
I'll check on that.